master/refs.html

```diff
@@ -19,15 +19,15 @@ <h1>References</h1>
 <h2 id="NormativeReferences">Normative references</h2>
 
 <dl class="references">
-  <dt id="ref-graphics-aria-1.0" class="normref">[<a href="https://www.w3.org/TR/graphics-aria-1.0/">graphics-aria-1.0</a>]</dt>
```

Comment:
did this get deleted, or moved to informative?

Reply:
ah okay moved, nevermind

Reply:
Yeah, there were a few refs entries that were out of order. Probably should have made it a separate commit, but it's hard to add new entries to an alphabetical list if the starting list isn't sorted!